### PR TITLE
doc: remove the instruction for installing cmake, ninja which is not …

### PR DESCRIPTION
**Pull request by Mia0451**
Hello,
I cleanup the installation guide related with installing cmake, ninja in building robolectric.

### CODE REVIEW

File: building-robolectric.md [hunk 1]
```diff
@@ -77,7 +76,7 @@ Open an msys2 terminal by running the "MSYS2 MINGW64" shortcut. This will ensure
 ```
 pacman -Syu # Update system
 pacman -Sy base-devel mingw-w64-x86_64-toolchain # Install the ming-w64-x86_64 package group
-pacman -Sy git mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja
+pacman -Sy git # Install git
 git clone --recurse-submodules https://github.com/robolectric/robolectric.git
```
Comment (hoisie) [inline building-robolectric.md:80]: rm `--recurse-submodules`

File: building-robolectric.md [hunk 2]
```diff
@@ -54,7 +54,6 @@ To build Robolectric on a Mac, [homebrew](https://brew.sh/) is required.
 ### Building
 
 ```
-brew install cmake ninja
 git clone --recurse-submodules https://github.com/robolectric/robolectric.git
```
Comment (hoisie) [inline building-robolectric.md:57]: rm `--recurse-submodules`

File: building-robolectric.md
Comment (hoisie) [inline building-robolectric.md:1]: Can you also remove the paragraph starting with `If you encounter `configure: error: Namespace support is required to build ICU``

File: building-robolectric.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ This section contains instructions for an Ubuntu Linux system. Other Linux syste
 ```
 # Install prerequisites
 sudo apt-get update
-sudo apt-get install git default-jdk openjdk-11-jdk clang make cmake ninja-build
+sudo apt-get install git default-jdk openjdk-11-jdk clang make
```
[the remark this answers is not in the document]
Comment (Mia0451) [reply building-robolectric.md]: removed clang (and in other places in this file).

Concerned with jdk version, is it 17 preferred over 11 now?

File: building-robolectric.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ This section contains instructions for an Ubuntu Linux system. Other Linux syste
...
 git clone --recurse-submodules https://github.com/robolectric/robolectric.git
```
[the remark this answers is not in the document]
Comment (Mia0451) [reply building-robolectric.md]: done

File: building-robolectric.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ This section contains instructions for an Ubuntu Linux system. Other Linux syste
...
 # If you forgot --recurse-submodules, you can also run `git submodule update --init --recursive` from the 'robolectric' directory.
```
[the remark this answers is not in the document]
Comment (Mia0451) [reply building-robolectric.md]: done

File: building-robolectric.md (hunk 2 above)
Comment (Mia0451) [reply building-robolectric.md]: done

File: building-robolectric.md
Comment (Mia0451) [reply building-robolectric.md:1]: done

File: building-robolectric.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mia0451) [reply building-robolectric.md]: sorry, not quite understand "AGP will force us to use JDK17".

I just build robolectric by following the instructions in this file using my Windows machine, looks like using java11 will succeed
build_robolectric

**Comment by hoisie** [comment]
This is a good start, we can do further cleanup in other PRs.

File: building-robolectric.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mia0451) [reply building-robolectric.md]: Oh, I forgot to run the test.

Mind telling me which IDE you will use normally? IntelliJ or Android Studio? Looks like Android Studio default to bundle install a version of AGP? 

I will use the same IDE as you all and run the test

[the remark this answers is not in the document]
Comment (Mia0451) [reply building-robolectric.md]: Got it. Thanks so much :)

**Pull request merged**